### PR TITLE
changing ELB health check from /console to /api 

### PR DESCRIPTION
**Pull request by ryanrf**
to remove the dependency on the webconsole being up. This now checks that the `api` endpoint is accessible, which should be the case if the api server is 'up'. If not, the master should be considered 'down'.

### CODE REVIEW

File: templates/openshift-master.template [hunk 1]
```diff
@@ -277,14 +277,14 @@ Parameters:
...
     Type: Number
-    AllowedValues: [ '3' ]
+    MinValue: '3'
   NumberOfEtcd:
     Default: '3'
-    Description: This Deployment requires 3 OpenShift Etcd instances
+    Description: This Deployment requires at least 3 OpenShift Etcd instances
     Type: Number
-    AllowedValues: [ '3' ]
+    MinValue: '3'
```
Comment (jaymccon) [inline templates/openshift-master.template]: Etcd should allow odd numbers only. 

Setting this to 4 could be particularly dangerous, as etcd needs 51% for quorum, if 2 instances are in the same AZ and that az goes down. Perhaps set allowed values to 3, 5, 7, 9, 11. If a customer needs more they can either fork the template or adjust the asg out of band.

Comment (pierreozoux) [reply templates/openshift-master.template]: Can we actually allow 1?
I'd like to be able to deploy small env without HA.

**Comment by ryanrf** [comment]
@jaymccon In addition to doing a curl to the master ELB address (on port `443` using the `/api` endpoint), I've also updated my environment to use that health check:
screen shot 2018-09-19 at 4 59 21 pm
screen shot 2018-09-19 at 4 59 34 pm

My most recent commit also addresses the requirement of having odd numbers of etcd (and masters) greater than 2 by using a regex, but I had to change the parameter type from number to string. The Cloudformation ASG resource requires the desired number and max number of instances to be a string anyway.

As for testing for this change, I've applied these settings to my own environment based on the quickstart templates, and I can confirm the regex works as expected.

**Comment by jaymccon** [comment]
Great stuff, thanks alot for this contribution @ryanrf . The change will get merged into develop branch and then ci will pick it up. Assuming all goes well with the ci tests it will be merged into master branch and the reference S3 bucket in a few hours.

**Pull request merged**